<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableView</class>
 <widget class="QDialog" name="TableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="1">
    <widget class="QTableView" name="tableView_2"/>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView_3"/>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tableView_4"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
